x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -71,46 +72,44 @@ export const getEuiDescriptionList = (
   host: HostItem | null,
   startDate: number,
   endDate: number
-): JSX.Element => {
-  return (
-    <EuiDescriptionList type="column" compressed>
-      {Object.entries(fieldTitleMapping).map(([field, title]) => {
-        const summaryValue: string | string[] | null = get(field, host);
-        return (
-          <React.Fragment key={field}>
-            <EuiDescriptionListTitle>{title}</EuiDescriptionListTitle>
-            {/*Using EuiDescriptionListDescription throws off sizing of Draggable*/}
-            <div>
-              {isArray(summaryValue)
-                ? summaryValue.map((value: string) =>
-                    createDraggable(value, field, startDate, endDate, dateFields)
-                  )
-                : createDraggable(summaryValue, field, startDate, endDate, dateFields)}
-            </div>
-          </React.Fragment>
-        );
-      })}
-    </EuiDescriptionList>
-  );
-};
+): JSX.Element => (
+  <EuiDescriptionList type="column" compressed>
+    {Object.entries(fieldTitleMapping).map(([field, title]) => {
+      const summaryValue: string | string[] | null = get(field, host);
+      return (
+        <React.Fragment key={`host-summary-${field}-${title}`}>
+          <EuiDescriptionListTitle>{title}</EuiDescriptionListTitle>
+          {/*Using EuiDescriptionListDescription throws off sizing of Draggable*/}
+          <div>
+            {isArray(summaryValue)
+              ? summaryValue.map((value: string) =>
+                  createDraggable(value, field, startDate, endDate, dateFields)
+                )
+              : createDraggable(summaryValue, field, startDate, endDate, dateFields)}
+          </div>
+        </React.Fragment>
+      );
+    })}
+  </EuiDescriptionList>
+);
 
 export const createDraggable = (
   summaryValue: string | null,
   field: string,
   startDate: number,
   endDate: number,
   relativeFields: string[]
-) => {
-  return summaryValue == null ? (
+) =>
+  summaryValue == null ? (
     <>{getEmptyValue()}</>
   ) : (
     <DraggableWrapper
-      key={summaryValue}
+      key={`host-summary-${field}-${summaryValue}`}
```

Comment:
Thanks for cleaning up the keys here -- this is definitely a spot where unique keys will matter since some of these description values can be arrays. 👍